cegs_portal/search/templates/search/v1/partials/_features.html

```diff
@@ -1,8 +1,18 @@
+
     <div>
         <table class="reg_data-table">
-            <tr><th>ID</th><th>Name</th><th>Feature Type</th><th>Cell Line</th><th>Location</th><th>Strand</th><th>Closest Gene</th><th>Reference Genome</th></tr>
+            <tr>
+                <th>ID</th>
+                <th>Name</th>
+                <th>Feature Type</th>
+                <th>Cell Line</th>
+                <th>Location</th>
+                <th>Strand</th>
+                <th>Closest Gene</th>
+                <th>Reference Genome</th>
+            </tr>
             {% for feature in features %}
-                <tr class="data-row">
+                <tr class="data-row" data-href="{% url 'search:dna_features' 'accession' feature.accession_id %}">
```

Comment:
does this need the javascript to set the click handler?

Reply:
The JavaScript partial is rendered on the reg_effect.html page. Its working well